sidebars.js

```diff
@@ -23,7 +23,10 @@ const sidebars = {
 			label: "Pricing & Limits",
 			type: "category",
 			link: { type: "doc", id: "pricing-limits/index" },
-			items: ["pricing-limits/free-plan-limits"],
+			items: [
+				"pricing-limits/free-plan-limits",
+				"pricing-limits/traffic-policy-unit-pricing",
```

Comment:
Looks like it's not showing up in the dropdown
<img width="232" height="69" alt="2025-08-20_10-33" src="https://github.com/user-attachments/assets/a3521fe5-5ffb-45a4-9c28-bc7113a876be" />

Reply:
Gotta wait for the preview to build